backend/src/main/java/com/celuveat/celuveat/restaurantlinkfetcher/infra/youtube/YouTubeRestaurantLinkFetcher.java

```diff
@@ -13,46 +15,59 @@ public class YouTubeRestaurantLinkFetcher implements RestaurantLinkFetcher {
     private final YouTubeDataApi youTubeDataApi;
 
     @Override
-    public List<String> fetchAllByChannelId(String channelId) {
+    public List<VideoHistory> fetchAllByCeleb(Celeb celeb) {
```

Comment:
```suggestion
    public List<VideoHistory> fetchAllVideoHistoryByCeleb(Celeb celeb) {
```
는 어때?

근데 지금 보니까 반환타입에 `VideoHistory`가 있어서 불필요해보이기도 하네 😅

Reply:
👍 `fetchAllVideoHistoriesByCeleb` 로 변경 완료!